back-end/routes/groups.js

```diff
@@ -46,6 +49,10 @@ router.put(
   add_to_pool.add_to_pool
 );
 
+
 router.put("/add_members/:group_id/:user_id", add_members.add_members)
+router.put("/add_to_ban/:group_id/:user_id", add_to_ban.add_to_ban)
```

Comment:
I'd recommended using middleware to get the user id, and asking for the bearer token in its place (refer to the add_to_pool route on lines 46-50). I know some of the other routes already ask for user_id, but I think this is actually important for security purposes, and in retrospect we should probably change the other routes. 

Just a note, if you use middleware to set the user_id, it will be stored under req.user_id, rather than req.params.user_id, so you'll have to make changes in the functions.